mesonbuild/mlog.py

```diff
@@ -30,10 +30,6 @@
     TV_Loggable = T.Union[str, 'AnsiDecorator', StringProtocol]
     TV_LoggableList = T.List[TV_Loggable]
 
-def is_windows() -> bool:
-    platname = platform.system().lower()
-    return platname == 'windows'
-
```

Comment:
We could also just make this a global constant just like `_in_ci`, no need to make it specifically be a class attribute on the logger.

Reply:
Same with all the rest, really.